editor/src/messages/portfolio/document/node_graph/node_properties.rs

```diff
@@ -995,27 +990,24 @@ pub fn query_assign_colors_randomize(node_id: NodeId, context: &NodePropertiesCo
 pub(crate) fn brightness_contrast_properties(node_id: NodeId, context: &mut NodePropertiesContext) -> Vec<LayoutGroup> {
 	use graphene_std::raster::brightness_contrast::*;
 
+	// Use Classic
+	let use_classic = bool_widget(ParameterWidgetsInfo::new(node_id, UseClassicInput::INDEX, true, context), CheckboxInput::default());
+
```

Comment:
creating a widget requires a mutable reference to the network interface to get the input type, although this will be changed in my next pr